cmd/server/main.go

```diff
@@ -1,3 +1,46 @@
 package main
 
-func main() {}
+import (
+	"log"
+	"net/http"
+
+	"github.com/Stern-Ritter/metrics-and-alerting-service/internal/config"
+	"github.com/Stern-Ritter/metrics-and-alerting-service/internal/storage"
+	handlers "github.com/Stern-Ritter/metrics-and-alerting-service/internal/transport"
+	"github.com/caarlos0/env"
+	"github.com/go-chi/chi"
+)
+
+func main() {
+	config, err := getConfig(config.ServerConfig{})
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	storage := storage.NewServerMemStorage()
+	run(&storage, config)
+}
+
+func run(storage *storage.ServerMemStorage, config config.ServerConfig) {
+	router := chi.NewRouter()
```

Comment:
SUGGESTION: it would be better if it's another function

Reply:
done